config/settings.yml

```diff
@@ -8,3 +8,6 @@ stacks:
 
 purl:
   iiif_manifest_url: 'https://purl.stanford.edu/%{druid}/iiif/manifest.json'
+zotero_api:
+  endpoint: 'https://api.zotero.org/%{type}s/%{id}'
+  solr_document_field: zotero_bibliography_ssi
```

Comment:
Yep, I'm not sure if there are any spotlight considerations with it being a text field? cc @jkeck ?